indigo/distribution.yaml

```diff
@@ -9137,16 +9137,16 @@ repositories:
     doc:
       type: git
       url: https://github.com/baalexander/rospy_message_converter.git
-      version: indigo-devel
+      version: master
     release:
       tags:
         release: release/indigo/{package}/{version}
-      url: https://github.com/jihoonl/rospy_message_converter-release.git
-      version: 0.3.0-2
+      url: https://github.com/baalexander/rospy_message_converter-release.git
```

Comment:
Well you probably cannot push to jihoonl's release repo, so that would give an error. I think a fork would be better, something like:

- delete https://github.com/baalexander/rospy_message_converter-release.git
- fork https://github.com/jihoonl/rospy_message_converter-release.git
- bloom-release -r indigo -t indigo --edit rospy_message_converter
 - (edit it to point to the master new branch, and anything else that has changed)
- close this pr, since that will open a new one